httpbp/client_middlewares.go

```diff
@@ -64,6 +64,7 @@ func NewClient(config ClientConfig, middleware ...ClientMiddleware) (*http.Clien
 	defaults := []ClientMiddleware{
 		MonitorClient(config.Slug),
 		Retries(config.MaxErrorReadAhead, config.RetryOptions...),
+		PrometheusClientMetrics(config.Slug),
```

Comment:
please update the doc comment of this function (which described what are the default middlewares added). same for other functions affected.